### PR TITLE
Edited in 5/21

### PR DESCRIPTION
**Pull request by naveen425**

### CODE REVIEW

**Comment by siddu1790** [comment]
+1

File: tedt1.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ dfdfdg
 sefds
 fdsf
 dsf
+siddu is here
```
Comment (siddu1790) [inline tedt1.txt]: review the coding standards

Comment (naveen425) [reply tedt1.txt]: can you please check agaiin

File: tedt1.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@ dfdfdg
...
 ds
 fds
```
Comment (siddu1790) [inline tedt1.txt]: xzcxc

**Pull request closed without merging**